osgp/shared/osgp-ws-core/src/main/resources/schemas/devicemanagement-ws-core.xsd

```diff
@@ -434,6 +434,21 @@
       <xsd:element name="LastCommunicationTime" type="xsd:dateTime" minOccurs="0" />
     </xsd:sequence>
   </xsd:complexType>
+
+  <xsd:complexType name="UpdatedDevice">
+    <xsd:sequence>
+      <xsd:element name="DeviceIdentification" type="common:Identification" />
```

Comment:
I was wondering, now that some fields that are not to be updated have been removed, why the device identification is part of `UpdatedDevice`.

A quick look into the code leads me to believe that this might lead to an undesirable outcome of some requests to update a device.
If the `UpdateDeviceRequest` has device identification A and the `UpdatedDevice` in that same request has a device identification B (not equal to A) this could lead to the `DeviceManagementEndpoint` logging it has received an update request for device A while it will go on to update device B instead (or not find a device B and throw an exception that the device does not exist).
You would expect (if one may change the identification at all) that device A would be updated to have identification B (if not used with another device already).

Perhaps the device identification should be removed from `UpdatedDevice` to avoid these issues.
Perhaps the code should be changed to do the correct update (if that should be possible).
An alternative could be to throw an exception if device identification A is not equal to B.